utils/containerd/Makefile

```diff
@@ -0,0 +1,83 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=containerd
+PKG_VERSION:=1.2.6
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/containerd/containerd/tar.gz/v${PKG_VERSION}?
+PKG_SOURCE_VERSION:=894b81a4b802e4eb2a91d1ce216b8817763c29fb
+PKG_HASH:=f2d578b743fb9faa5b3477b7cf4b33d00501087043a53b27754f14bbe741f891
+
+PKG_MAINTAINER:=Gerard Ryan <G.M0N3Y.2503@gmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+GO_PKG:=github.com/containerd/containerd
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/containerd/config
+config CONTAINERD_SECCOMP
+	depends on PACKAGE_containerd
+	bool "Enable support for seccomp in containerd"
+	default KERNEL_SECCOMP
+	help
+	  Build containerd with support for seccomp filters.
+	  Select libseccomp which also pulls-in the needed kernel features.
+endef
+
+define Package/containerd
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=containerd container runtime
+  URL:=https://containerd.io/
+  DEPENDS:=$(GO_ARCH_DEPENDS) @TARGET_x86_64 +btrfs-progs +runc +CONTAINERD_SECCOMP:libseccomp
```

Comment:
I kept getting a linking error when trying to remove these from the Makefiles. I've left these unchanged for now, but maybe you can see something I am missing in the latest changes?

Reply:
Is it something like "package xxx is missing dependencies for ..."?   It's from sanity check of dependencies by the build system.  Right now it can only track dependencies specified in `DEPENDS` variable.  We will need to preserve it here.

Reply:
Yeah, exactly.
```
Package runc is missing dependencies for the following libraries:
libseccomp.so.2
Makefile:88: recipe for target '/openwrt/bin/packages/x86_64/packages/runc_1.0.0-rc8-1_x86_64.ipk' failed
```